engines/director/sound.cpp

```diff
@@ -847,6 +847,14 @@ bool SNDDecoder::hasLoopBounds() {
 	return _loopStart != 0 || _loopEnd != 0;
 }
 
+bool SNDDecoder::hasValidLoopBounds() {
+	return hasLoopBounds() && _loopStart < _loopEnd  && _loopEnd <= _size;
```

Comment:
Please, remove extra space in from of the second `&&`

Reply:
You can probably tell from this that I am a hacky developer who relies on her linting tools too much. I'll try and configure this for next time!